Dockerfile

```diff
@@ -23,7 +23,11 @@ COPY api/ api/
 COPY cmd/ cmd/
 COPY controllers/ controllers/
 COPY pkg/ pkg/
+COPY charts/ charts/
+COPY config/ config/
+COPY manifests/ manifests/
```

Comment:
What about manifests at the build time? Can we remove those as well @pacevedom ?

Reply:
Looks like we don't need it, removed it.